processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -193,7 +199,8 @@ public DateTime getMaxIngestedEventTime()
       Interval interval,
       VirtualColumns virtualColumns,
       Granularity gran,
-      boolean descending
+      boolean descending,
+      @Nullable QueryMetrics<?> queryMetrics
```

Comment:
@drcrallen `null` queryMetrics is allowed _not_ for backward compatibility, it is valid, means that the caller doesn't want to record metrics. OTOH updated signatures are not in the interfaces/classes which are extension points, so adding "backwards compatible" deprecated methods is pointless.